pages/course/[courseId]/unit/HTML/1.tsx

```diff
@@ -0,0 +1,95 @@
+import Card from "../../../../../components/coding/Card";
+import Sidebar from "../../../../../components/coding/studentPortal/Sidebar";
+import ProgressBar from "../../../../../components/coding/studentPortal/ProgressBar";
+import Navbar from "../../../../../components/ui/Navbar";
+import { ResourceRow } from "../../../../../components/coding/studentPortal/ResourceRow";
+import { Button } from "../../../../../components/ui/Button";
+import Layout from "../../../../../components/coding/studentPortal/Layout";
+
+const HTML1 = () => {
+  return (
+    <>
+      <div className="space-y-4">
+        <div className="grid grid-cols-1 gap-x-8">
+          <div className="">
+            <ProgressBar completed={100} />
+            <h1 className="font-bold text-5xl">HTML</h1>
+            <div className="space-y-4 ">
+              <div className="flex flex-col space-y-4 mt-12">
+                <p>
```

Comment:
not sure if you made any real changes to this, or if we can just delete this page. The other copy uses the LessonComponent list pattern, which is preferable